FAQ.md

```diff
@@ -0,0 +1,98 @@
+#Frequently Asked Questions
```

Comment:
Seems to work fine here: https://github.com/jeffmo/ecma262/blob/master/FAQ.md
Will add spaces if you prefer, though